registry/root.go

```diff
@@ -75,7 +77,7 @@ var GCCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
-		err = storage.MarkAndSweep(ctx, driver, registry, dryRun)
+		err = storage.MarkAndSweep(ctx, driver, registry, dryRun, manifestswithouttags)
```

Comment:
I tried to find way to do that, but could not find anything that works. Can you help me little bit where I should define that struct (it needs to be available in other files where markandsweep is referenced)